app/services/course/assessment/reminder_service.rb

```diff
@@ -16,7 +16,8 @@ def closing_reminder(assessment, token)
     # Send reminder emails to each student who hasn't submitted.
     recipients = uncompleted_students(assessment)
     recipients.each do |recipient|
-      Course::Mailer.assessment_closing_reminder_email(assessment, recipient).deliver_later
+      # Need to get the User model from the Course User because we need the email address.
+      Course::Mailer.assessment_closing_reminder_email(assessment, recipient.user).deliver_later
```

Comment:
You'll have to change the method documentation for `assessment_closing_reminder_email`.